### PR TITLE
Adds syndicate balloon mood modifier for antagonists

### PR DESCRIPTION
**Pull request by ShizCalev**
You won't just look badass, you'll **FEEL** badass.

reasoning: holding the balloon attracts tons of attention, and most players will assume you're a traitor and will watch you like a fucking hawk and jump you the second you give them a reason. Might as well give a little more incentive to show off just how much you don't give a fuck.

:cl: ShizCalev
add: An antagonist holding a syndicate balloon will now receive a positive mood modifier.
/:cl:

### CODE REVIEW

File: code/datums/mood_events/generic_positive_events.dm [hunk 1]
```diff
@@ -52,6 +52,11 @@
 	mood_change = 12
 	hidden = TRUE
 
+/datum/mood_event/badass_antag
+	description = "I'm a fucking badass and everyone around me knows it. Just look at them; they're all fucking shaking at the mere thought of me around."
+	mood_change = 15
+	hiiden = TRUE
```
Comment (Barhandar) [inline code/datums/mood_events/generic_positive_events.dm]: Typo, should be "hidden".

**Comment by evsey9** [comment]
Pretty sure bad moodifiers can still affect them.

**Comment by BeeSting12** [comment]
Antagonists always have a positive mood so theres no real point.

**Comment by ExcessiveUseOfCobblestone** [comment]
remove the antag moodlet that seems stupid.

**Comment by Nabski89** [comment]
If you die and are cloned you lose the "I have an objective" positive mood.

**Comment by Kagus** [comment]
Traitors have a slight mood boost that makes functioning without constant mood pampering more possible. They're still entirely capable of getting saddened by the miserable state of most shitstain stations though... Unlike cultists, who have an absolutely hysterical +40 modifier for being in the loving grasp of Nar'Sie.

While I'm definitely for retaining the uselessness of the balloon, I feel like this PR doesn't really tip the scales enough in either direction to really make much of a difference.

And, I mean... It's a balloon for chrissakes. Balloons are fuckass awesome.

**Comment by ExcessiveUseOfCobblestone** [comment]
That just seems like a handwave for not wanting to flesh out and/or properly balance the feature.

If the feature is so insufferable you need to be able to ignore it completely to actually do anything antagonistic it's probably not very fun for nonantagonists either.

**Comment by Tlaltecuhtli** [comment]
cool now i can antag test people by looking at their hp before they crit

**Comment by ExcessiveUseOfCobblestone** [comment]
You'd be taking quite a chance since having your mood up isn't antag-exclusive lol.

**Comment by Qustinnus** [comment]
@ExcessiveUseOfCobblestone the antag mood is there for the people that bitched about antags not caring about their conditions due to being trained killers or ecstatic zealots

**Comment by ExcessiveUseOfCobblestone** [comment]
lame

**Review by ninjanomnom — DISMISSED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**